Expensify/App
# [No QA] add "coming soon" page to Expensify Help site

~HOLD ON https://github.com/Expensify/App/pull/9765~

cc @roryabraham @michelle-thompson 

### Details

This PR replaces the main page with a "coming soon" page. We're going to replace it again with the main page once we upload more content to the Expensify Help site. 

Also this PR is branched off from https://github.com/Expensify/App/pull/9765. Once https://github.com/Expensify/App/pull/9765 is merged I'm going to merge main into this branch and the real number of commits will be displayed.

### Fixed Issues

$ https://github.com/Expensify/App/issues/9830

### Tests

0. (you need ruby and bundler installed to do this, more info [here](https://github.com/Expensify/App/tree/main/docs#pre-requisites))
1. Run:

    ```bash
    cd docs
    bundle install
    bundle exec jekyll serve
    ```
2. Make sure the "coming soon" page is displayed in the root directory (index.html). 
3. Verify you can access the main site by visiting http://localhost:4000/main (atm the content area of the main page is empty, it just displays the LHN).

### Screenshots


#### Web

![localhost_4000_](https://user-images.githubusercontent.com/6829422/178615398-ea8f64ce-dbc0-4e01-8124-bb836e3a82d2.png)

#### Mobile Web

<img src="https://user-images.githubusercontent.com/6829422/178615405-2e1ce818-53a4-4d31-8c78-3d6bfe3b49b1.png" width="400">
